src/app/containers/Caption/index.jsx

```diff
@@ -9,22 +9,32 @@ import InlineLink from '../InlineLink';
 
```

Comment:
It seems like you haven't pushed your Storybook changes that you had locally to the branch. Can you please push these?